docs/blog/posts/2023/sr-linux-kubernetes-anycast-lab.md

```diff
@@ -0,0 +1,955 @@
+|  6 | cli3         | 800f2389d7b0 | ghcr.io/hellt/network-multitool                                                                             | linux         | running | 172.20.20.10/24 | 2001:172:20:20::a/64 |
+|  7 | cli4         | 9049f6031e81 | ghcr.io/hellt/network-multitool                                                                             | linux         | running | 172.20.20.11/24 | 2001:172:20:20::b/64 |
+|  8 | leaf1        | 72b35012bfe4 | ghcr.io/nokia/srlinux:23.3.3                                                                                | srl           | running | 172.20.20.2/24  | 2001:172:20:20::2/64 |
+|  9 | leaf2        | afcd88f998d5 | ghcr.io/nokia/srlinux:23.3.3                                                                                | srl           | running | 172.20.20.6/24  | 2001:172:20:20::6/64 |
+| 10 | leaf3        | ba4ac1d2b0cf | ghcr.io/nokia/srlinux:23.3.3                                                                                | srl           | running | 172.20.20.8/24  | 2001:172:20:20::8/64 |
+| 11 | leaf4        | eb04640e81bb | ghcr.io/nokia/srlinux:23.3.3                                                                                | srl           | running | 172.20.20.3/24  | 2001:172:20:20::3/64 |
+| 12 | spine1       | 28adfba1fb86 | ghcr.io/nokia/srlinux:23.3.3                                                                                | srl           | running | 172.20.20.9/24  | 2001:172:20:20::9/64 |
+| 13 | spine2       | 6461294e19fb | ghcr.io/nokia/srlinux:23.3.3                                                                                | srl           | running | 172.20.20.4/24  | 2001:172:20:20::4/64 |
++----+--------------+--------------+-------------------------------------------------------------------------------------------------------------+---------------+---------+-----------------+----------------------+
+
+```
+
+## Minikube MetalLB installation
+
+As are using MetalLB, first we need to enable it in the Minikube cluster:
+
+```bash
+minikube addons enable metallb -p cluster1
+```
+
+MetalLB has two [modes of operation](https://metallb.universe.tf/concepts/): Layer2 and BGP. For this Lab we will use BGP. 
+
+MetalLB also provides two different [BGP implementations](https://metallb.universe.tf/concepts/bgp/):
+
+1. Native
+2. FRR (provides BGP sessions with BFD support)
+
+We will use the *FRR* implementation. We install it with the following command:
+
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/main/config/manifests/metallb-frr.yaml
+```
+
+
+## Leaf/Spine fabric verification
+
+Because we used startup configuration files for all of the switches of our lab, the BGP peering between Leaf/Spine switches should be working by now:
```

Comment:
I think here we should be more verbose. You haven't explained what is deployed "service-wise" in the fabric.
If it is L2 or L3 EVPN, then we need to show with a diagram (or verbally), what readers should expect to get at this point. Like which nodes should have reachability and why we need ip-vrf-1.

Currently we jump to fabric verification process, but we haven't cleared what exactly we're verifying and why this service is configured

Reply:
added diagram and anycast-gw explanation